public/js/reducers/breakpoints.js

```diff
@@ -138,6 +134,26 @@ function update(state = State(), action: Action) {
   return state;
 }
 
+function toggleBreakpoints(state: any, action: any) {
```

Comment:
That is weird indeed. Unfortunately I don't see why the type inference doesn't work here 😞